src/components/TeamBio.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Image from 'react-bootstrap/Image'
+
+class TeamBio extends React.Component {
+    render() {
+        const { pic, name, role, bio } = this.props.tmData;
+
+        return (
+            <>
+                <Image src={ pic } fluid></Image>
+                <p>{ name }</p>
+                <p>{ role }</p>
+                { bio.split('\n').map((pg, idx) => {
+                    return <p key={ idx }>{ pg }</p>
+                })}
```

Comment:
This works fine, but you can also achieve similar with CSS. 

`<p style={{ whiteSpace: "pre-wrap" }}>`

This will honor the `\n` as a new line. You can do a double `\n\n\` to space out paragraphs. Up to you, just thought I'd mention it.